Makefile

```diff
@@ -125,7 +128,8 @@ help:
 	@echo -e "\033[1mFORMATING, LINTING AND TESTING TOOLS TARGETS\033[0m "
 	@echo "- lint/autolint      Python code quality assurance"
 	@echo "- shell              Pylons shell (for debugging)"
-	@echo "- test               Functional and integration nose tests"
+	@echo "- test-functional    Functional nose2 tests"
+	@echo "- test-integration   Integration nose2 tests"
```

Comment:
Keep also the `test` target in the help.

Reply:
Good idea!